source/support/connect-to-govwifi-using-a-windows-device.html.erb

```diff
@@ -0,0 +1,134 @@
+---
+title: Connect to GovWifi using a Windows device - GovWifi
+---
+
+<div class="container">
+  <div class="grid-row">
+    <div class="column-one-third">
+      <%= partial '/shared/support/_sidebar' %>
+    </div>
+
+    <div class="column-two-thirds column-minimum">
+      <h1 class="heading-large">Connect a Windows device</h1>
+      <h2 class="heading-medium">Instructions</h2>
+      <h3 class="heading-small">Contents</h3>
+
+      <ul class="list">
+        <li><%= link_to "Connect using Windows 8 or 10", "#connect-windows-8", class: "govuk-link" %></li>
+        <li><%= link_to "Support for Windows 8 and 10", "#support-windows-8", class: "govuk-link" %></li>
+        <li><%= link_to "Connect using Windows 7", "#connect-windows-7", class: "govuk-link" %></li>
+        <li><%= link_to "Support for Windows 7", "#support-windows-7", class: "govuk-link" %></li>
+      </ul>
+
+      <p>Different versions of Windows work in slightly different ways. If you don't know which version of Windows your device is running, read <%= link_to "Microsoft's guidance", "https://support.microsoft.com/en-gb/help/13443/windows-which-operating-system" %>.</p>
+
+      <h3 id="connect-windows-8">Connect using Windows 8 or 10</h3>
```

Comment:
Do we need this `id`? There are quite a few `ids` on this page. And do we need `class="heading-small"`? We use it elsewhere

Reply:
Yeah there are anchor links pointing at #connect-windows-8.
The id itself should be unique though.

Without the class heading small an H3 is the same size as an H2